MODULE.bazel

```diff
@@ -13,9 +13,6 @@ bazel_dep(name = "rules_license", version = "0.0.8")
 bazel_dep(name = "rules_pkg", version = "0.10.1")
 bazel_dep(name = "rules_python", version = "0.31.0")
 
-non_module_deps = use_extension("//:repositories.bzl", "non_module_deps")
```

Comment:
perhaps rules_python dependency can be dropped as well? It seems it's only needed by io_bazel and not referenced in this repo